lib/compat.php

```diff
@@ -418,6 +418,23 @@ function gutenberg_render_block_with_assigned_block_context( $pre_render, $parse
 		}
 	}
 
+	if ( isset( $wp_query->tax_query->queried_terms['post_tag'] ) ) {
+		$context['query'] = array( 'tagIds' => array() );
```

Comment:
This is not correct and I'll have to change it. I'm investigating first something in the loop.